stringResources/en-US/resources.resjson

```diff
@@ -1,32 +1,26 @@
 {
-	"Visual_Groups": "Groups",
-	"Visual_Group": "Group",
-	"Visual_Values": "Values",
-	"Visual_ShowCategoryLabel": "Show category label",
-	"Visual_ShowDataLabels": "Show data labels",
-	"Visual_FontSize": "Font size",
-	"Visual_TopCategoryColor": "Top category color",
-	"Visual_DisplayLegendOptions": "Legend",
-	"Visual_Show": "Show",
-	"Visual_Top": "Top",
-	"Visual_Bottom": "Bottom",
-	"Visual_Left": "Left",
-	"Visual_Right": "Right",
-	"Visual_TopCenter": "Top Center",
-	"Visual_BottomCenter": "Bottom Center",
-	"Visual_LeftCenter": "Left Center",
-	"Visual_RightCenter": "Right Center",
-	"Visual_Position": "Position",
-	"Visual_Title": "Title",
-	"Visual_LegendName": "Legend Name",
-	"Visual_Color": "Color",
-	"Visual_TextSize": "Text Size",
-	"Visual_Tooltip": "Tooltip",
-	"Visual_DisplayUnits": "Display Units",
-	"Visual_Precision": "Decimal Places" ,
-	"Visual_Description_Legend": "Display legend options",
-	"Visual_Description_Position": "Select the location for the legend",
-	"Visual_Description_Title": "Display a title for legend symbols",
-	"Visual_Description_LegendName": "Title text"
-
+    "Visual_Groups": "Groups",
```

Comment:
Is that change mandatory for your PR?
If not, please revert it

Reply:
I need to add new localizations for the new inputs and while doing so, I removed the unused localizations and extraneous whitespace. I was asked to rename the category displayname to specify "selected category" in [this comment](https://github.com/microsoft/powerbi-visuals-sunburst/pull/94#issuecomment-1617683260). I believe all the changes are necessary. I've added back the tabs instead of 4-space indents to make the git diff show up nicer: [58ac23c](https://github.com/microsoft/powerbi-visuals-sunburst/pull/95/commits/58ac23ca331711844c387f6ad5e4e19bdb717976)